### DISCUSSION TITLE
Student developer pack - copilot not active

### DISCUSSION THREAD

**Discussion by Peter-Hu12138** [community/community#145995] [GitHub Education] [answered]
### Hi everyone,

Hi everyone, my student-ship has been approved on 20th, Oct, but I cannot follow [free access to copilot](https://docs.github.com/en/copilot/managing-copilot/managing-copilot-as-an-individual-subscriber/managing-your-copilot-subscription/getting-free-access-to-copilot-as-a-student-teacher-or-maintainer) to get the access to copilot since there is nothing on "my copilot" page as depicted in the URL doc. What should I do? Are there any ways I can check my student dev pack status?

Thanks in advance for your help.

**Comment by davevad93** [comment, marked as answer]
Hi @Peter-Hu12138 ,

I guess your status is "`Congrats! Your request for @Peter-Hu12138  was approved and your benefits are now available.`". Check if you can redeem your offer by going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue attaching screenshots as well.

**Reply by Peter-Hu12138** [reply]
It works; thank you! By the way, does it mean I should always use "All offers" to redeem my benefits, or at least see it as the first option to go with?

**Reply by davevad93** [reply]
You're welcome @Peter-Hu12138 , normally to redeem `Copilot` you have to go to [your copilot](https://github.com/settings/copilot) but sometimes it doesn`t work, so another option is to go to the “***All offers***” section on the [pack offers page](https://education.github.com/pack) which always works so I recommend it as the first option to redeem student benefits.
